app/src/main/java/com/zulip/android/networking/GravatarAsyncFetchTask.java

```diff
@@ -47,7 +48,7 @@ public static URL sizedURL(Context context, String url, float dpSize) {
         float px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP,
                 dpSize, r.getDisplayMetrics());
         try {
-            return new URL(url + "&s=" + px);
+            return new URL(UrlHelper.addHost(url)  + "&s=" + px);
```

Comment:
@vishwesh3 It would be better to have a check the URL is incomplete then only add the host in there. It would be great if you could do it really soon, want to get this working really soon :)